include/ginkgo/core/matrix/sellp.hpp

```diff
@@ -258,8 +258,8 @@ class Sellp : public EnableLinOp<Sellp<ValueType, IndexType>>,
     /**
      * @copydoc Sellp::val_at(size_type, size_type, size_type)
      */
-    value_type val_at(size_type row, size_type slice_set, size_type idx) const
-        noexcept
+    value_type val_at(size_type row, size_type slice_set,
+                      size_type idx) const noexcept
```

Comment:
I think the formatting is off. Maybe you have a different `clang-format` version?